experiment/assignment.md

```diff
@@ -1,8 +1,8 @@
-1. Design the full adder circuit as given in the theory section. Save this full adder circuit so that it can later be imported as a black box, which takes 3 inputs (A, B, and C) and produces 2 outputs (Sum and Carry)
-
-2. Import Full Adder circuits designed in 1st part and construct a 4 BIT binary adder using 4 full adders. Given two 4 bit binary numbers a3a2a1a0 and b3b2b1b0, the circuit should output s3s2s1s0 and c-out which is the final carry. For circuit diagram, refer to the theory section
-
-3. Design a 3 bit full subtractor using EX-OR, AND, OR and NOT gates. Given X, Y and Z, the circuit should output D (difference) and B (Borrow). D = X (ex-or) Y (ex-or) Z and B = X'.Y + Y.Z + X'.Z
-
-4. Can the Final OR gate in the Full Adder circuit (given in the theory section) be replaced by an EX-OR gate ? If yes why? If no Why not?
-
+1. Design the full adder circuit as given in the theory section. Save this full adder circuit so that it can later be imported as a black box, which takes 3 inputs (A, B, and C) and produces 2 outputs (Sum and Carry)
+
+2. Import Full Adder circuits designed in 1st part and construct a 4 BIT binary adder using 4 full adders. Given two 4 bit binary numbers a3a2a1a0 and b3b2b1b0, the circuit should output s3s2s1s0 and c-out which is the final carry. For circuit diagram, refer to the theory section
+
+3. Design a 3 bit full subtractor using EX-OR, AND, OR and NOT gates. Given X, Y and Z, the circuit should output D (difference) and B (Borrow). D = X (ex-or) Y (ex-or) Z and B = X'.Y + Y.Z + X'.Z
+
+4. Can the Final OR gate in the Full Adder circuit (given in the theory section) be replaced by an EX-OR gate ? If yes why? If no Why not?

```

Comment:
Remove capitalization from the Why in 'If no Why not?'

Reply:
Done